DSCResources/cChocoInstaller/cChocoInstaller.psm1

```diff
@@ -229,7 +232,7 @@ function InstallChoco
     
     # unzip the package
     Write-verbose "Extracting $file to $tempDir..."
-    Start-Process "$7zaExe" -ArgumentList "x -o`"$tempDir`" -y `"$file`"" -Wait
+    Start-Process "$7zaExe" -ArgumentList "x -o`"$tempDir`" -y `"$file`"" -Wait -NoNewWindow
```

Comment:
I just wasn't sure if there was some reason on this NOT to include the output. It gets a bit chatty and not in a verbose logging kind of way. :)